src/style.css

```diff
@@ -0,0 +1,126 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500&family=Roboto:wght@400;500;700&display=swap');
+
+.nav-top {
+  background-color: black;
+  height: 900px;
+}
+
+body {
+  border-top: 30px solid black;
+  width: 100%;
+  height: 900px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+  font-size: 40.5%;
+  background-color: #f1f2f5;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.main-container {
+  width: 27%;
+  font-size: 2em;
+  background-color: rgba(250, 250, 250, 0.767);
+  box-shadow: 1px 1px 4px 1px #aaa;
+  margin-top: -400px;
+}
+
+.main-container .container {
+  width: 100%;
+  padding: 10px;
+}
+
+.main-container .container .container-header {
```

Comment:
[optional] You can improve the readability of your code by using the shorter class selector. For example, instead of using the chained class name` .main-container .container .container-header,` you can use the class name `.container-header`. This will make your code easier to read and understand.

> Please do the same logic for CSS selector at `line 45`, `line 54`, `line 62`, `line 67`, `line 72`,  `line 77`, `line 90`, `line 95`, `line 99`,